webui/react/src/pages/WorkspaceList.tsx

```diff
@@ -94,7 +93,7 @@ const WorkspaceList: React.FC = () => {
     settings.tableOffset,
     settings.user ]);
 
-  usePolling(fetchWorkspaces, { rerunOnNewFn: true });
+  usePolling(fetchWorkspaces);
```

Comment:
we need to manually rerun when the parameters update in order to set the loading state properly. therefore this duplicates